tasks/10.html

```diff
@@ -12,6 +12,15 @@ <h1 class="global-heading">&nbsp;</h1>
 
         <main class="global-results">
             <style>
+                p.title::first-letter
```

Comment:
В селекторе класса тег указывать не обязательно. Ниже - аналогично
```suggestion
                .title::first-letter
```